neard/src/runtime/mod.rs

```diff
@@ -144,9 +144,10 @@ impl NightshadeRuntime {
         genesis: &Genesis,
         initial_tracking_accounts: Vec<AccountId>,
         initial_tracking_shards: Vec<ShardId>,
+        trie_viewer_state_size_limit: Option<u64>,
```

Comment:
Just wanted to share my pain: I really struggle when we keep dropping new arguments to the methods, and soon end up with `new(1, None, Some(3), "node.test", 42, "", None)` :cry: 

Well, we are already there:

```
neard::NightshadeRuntime::new(
        Path::new("."),
        store1,
        &genesis,
        vec![],
        vec![],
        None,
    )
```

It will require another round of refactoring later outside of this PR, I believe, I just wanted to bring it up for future reference.

Reply:
:+1: here's the pattern we use in rust-analyzer to fight this problem: https://github.com/rust-analyzer/rust-analyzer/blob/master/docs/dev/style.md#functions-with-many-parameters